### PR TITLE
Run the update ci job less frequently

### PR DESCRIPTION
**Pull request by daviddavis**
[noissue]

### CODE REVIEW

**Review by fao89 — APPROVED** [review]

**Comment by fao89** [comment]
FYI @rochacbruno

File: templates/github/.github/workflows/update_ci.yml.j2 [hunk 1]
```diff
@@ -5,8 +5,8 @@ on:
   {%- if sync_ci %}
   schedule:
     # * is a special character in YAML so you have to quote this string
-    # runs at 2:30 UTC daily
-    - cron: '30 2 * * *'
+    # runs at 2:30 UTC every 3 days
```
[the remark this answers is not in the document]
Comment (mdellweg) [reply templates/github/.github/workflows/update_ci.yml.j2:8]: I was just about to suggest the same.

**Comment by quba42** [comment]
What about making this configurable by plugin template?

**Comment by daviddavis** [comment]
I don't know that most plugins would want to tweak the configuration so I'm not sure it's worth the added complexity. That said, I'm happy to change this if anyone disagrees.

**Comment by daviddavis** [comment]
My initial thought was to have a single setting both control if the job runs and what the cron schedule is but this would be difficult since plugins have already have set the `sync_ci` setting. So alternatively we could introduce a second setting (`sync_ci_schedule`?) and this would be easy but it adds a new setting.

Or we could of course just merge this and wait for plugins to ask for the ability to configure the setting. 

I don't have a strong opinion either.

**Comment by daviddavis** [comment]
I'll merge this but we can continue the discussion about how to make the cron schedule configurable which I'll address in a follow up PR.

**Pull request merged**